cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -47,6 +50,47 @@ param (
 $app_name = '{{ app_name }}'
 $app_version = '{{ app_version }}'
 
+$install_path = $null
+# If you're reading this install script generated by an app, and
+# this looks strange to you, it's because it's possible for there
+# to be multiple generated paths here. If you're only seeing one
+# $install_path assignment, it's because the app you're looking at
+# has been configured with a single install path.
```

Comment:
I don't understand what this comment is trying to say.

Regardless, this implementation has the same issues I describe in the shell impl.

Reply:
Take a look at the generated script this comment is hanging off: https://github.com/axodotdev/cargo-dist/pull/947/files#diff-dd947bc4ad8e756b1f8021216e9c9888f3e1635b526b7bfb289febdad0c6b889R54

If there's *only one* path set, then you get a weird looking condition where we assign an empty string, immediately check if that string is empty and do something, then after that check again if the string is empty. This only looks "normal" in scripts with multiple strategies assigned. It serves as an explanation that there's a reason for this to make sense even if the specific generated file they've seen looks odd.